test/utils.py

```diff
@@ -65,10 +65,14 @@ def wait_file(dir_name, file_pattern, exclude_list, timeout_sec=1):
     iter_count = 0
 
     while True:
-        files = os.listdir(dir_name)
-        for file in files:
-            if re.match(file_pattern, file) is not None and file not in exclude_list:
-                return file
+        try:
```

Comment:
It looks like it could be moved to a separate commit with an explanation of the changes in the commit message.

Reply:
Moved to separate commit.